src/pushInfo/__snapshots__/PushInfo.unit.tsx.snap

```diff
@@ -41,18 +39,16 @@ exports[`Should also have the correct icon. 1`] = `
 
 exports[`Should have the correct attributes and text. 1`] = `
 <div
-  className="jsx-3382816266 kirk-pushInfo"
+  className="kirk-pushInfo"
```

Comment:
For snapshot testing, use the styled version, it will also display the associated CSS (which will help for diffs).

Reply:
Done